### ISSUE TITLE
Javascript errors since 1.16.4 release

### ISSUE THREAD

**Issue by DBradleyRI** [xanderfrangos/twinkle-tray#905]
I receive a series of Javascript errors since the recent release. Attaching one now and will screenshot others when they return in case they help decipher the issue further.

Running Windows 11.

![twinkle-tray-error](https://github.com/user-attachments/assets/b8bf2760-3ec0-45bc-b318-cac189ca356a)

**Comment by Napay1** [comment]
I have the same issue, it has been a week already

![image](https://github.com/user-attachments/assets/b4bef9a9-1e95-4fca-9408-43a4679e2349)

And when I click "Aceptar", then this:

![image](https://github.com/user-attachments/assets/faf17c12-9e4c-4d45-8296-a78b46a19f49)

Please fix

**Comment by DBradleyRI** [comment]
Turned on WMI-Bridge (the other four were already turned on also), restarted the program, and still had the issue unfortunately. Anything else I can provide to help troubleshoot?

**Comment by xanderfrangos** [comment]
@DBradleyRI To clarify, I meant that you should set all of those to "Off", except "WMI-Bridge", then restart Twinkle Tray. Was that what you did?

**Comment by DBradleyRI** [comment]
@xanderfrangos no, I did not. I just made that change and restarted it. The error seems to appear after not using the computer and then waking it again, so I'll give it a day or two and update you. Thank you.
